tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cpp

```diff
@@ -11,6 +11,9 @@
 #include <glib.h>
 #include <glib/gstdio.h>        /* GStatBuf */
 #include <nnstreamer_plugin_api_filter.h>
+#include <nnstreamer.h>
+#include <nnstreamer-single.h>
+#include <nnstreamer-capi-private.h>
```

Comment:
If you are going to test with Tizen ML APIs, ```nnstreamer-capi-private.h``` shouldn't be required.

Reply:
'nnstreamer-capi-private.h' is included to use 'set_feature_state'.
'set_feature_state' need to construct the pipeline.

Reply:
If that's all you use, then, it's perfectly fine :)